tests/integration/targets/dcnm_links/tests/dcnm/dcnm_links_misc.yaml

```diff
@@ -0,0 +1,900 @@
+##############################################
+##               SETUP                      ##
+##############################################
+
+- name: Remove local log file
+  local_action: command rm -f dcnm_links.log
+
+- block:
+
+#############################################
+#               DELETE                     ##
+#############################################
+
+    - name: Initial setup - Delete Links on numbered fabric
+      cisco.dcnm.dcnm_links: &links_delete
+        state: deleted                                           # choose from [merged, replaced, deleted, query]
+        src_fabric: "{{ ansible_num_fabric }}"
+        config:
+          - dst_fabric: "{{ ansible_num_fabric }}"               # Destination fabric
```

Comment:
Remind me, does this module delete all links if state is deleted and we omit the config block from the playbook?

Reply:
No, config block is mandatory for Links Delete. This is because the system by default has links when it comes up and if the user by mistake executes a delete with no config block, then it will result in all links being deleted which may not be desirable. So to avoid this we force the user to explicitly include the config block so that only the matching links are deleted